packages/core/useRefHistory/index.ts

```diff
@@ -141,21 +141,23 @@ export interface UseRefHistoryReturn<Raw, Serialized> {
  * @see https://vueuse.org/useRefHistory
  * @param source
  * @param options
+ * @param eventFilter optional
  */
 export function useRefHistory<Raw, Serialized = Raw>(
   source: Ref<Raw>,
   options: UseRefHistoryOptions<Raw, Serialized> = {},
+  eventFilter: EventFilter = bypassFilter,
```

Comment:
We should have the `eventFilter` inside the options.

Reply:
updated